docs/update-locking.md

```diff
@@ -2,7 +2,7 @@
 
 Locking updates means that the Supervisor will not be able to kill your application. This is meant to be used at critical sections of your code where you don't want to be interrupted, or to control that updates are only installed at certain times.
 
-In order to do this, users can create a file called `resin-updates.lock` that will prevent the Supervisor from killing and restarting the app. As any other lockfile, the Supervisor itself will create such file before killing the app, so you should only create it in exclusive mode. This means: only create the lockfile if it doesn't already exist. Several tools exist to simplify this process, for example [npm/lockfile](https://github.com/npm/lockfile).
+In order to do this, users create a lockfile called `resin-updates.lock` in a way that it has exclusive access, which will prevent the Supervisor from killing and restarting the app. As any other lockfile, the Supervisor itself will create such file before killing the app, so you should only create it in exclusive mode. This means: only create the lockfile if it doesn't already exist. The exclusive access is achived by opening the lockfile with the [O_EXCL and O_CREAT flags](https://linux.die.net/man/3/open), and several tools exist to simplify this process with examples given [below][#creating-the-lockfile].
```

Comment:
NIT
- `users create a lockfile` --> `users can create a lockfile`
- `As any other lockfile` --> `As with any other lockfile`
- `This means: only create the lockfile if it doesn't already exist.` --> `This means that the lockfile should only be created if it doesn't already exist`
- `The exclusive access is achived` --> `The exclusive access is achieved`
- `[below][#creating-the-lockfile]` This doesn't appear to render correctly when viewing the document.

Reply:
Updated, thanks!

Reply:
BTW, the `below` will work in the docs, when it is pulled in, the headers are rendered with anchors; not so in just markdown preview.